src/libraries/System.Text.Json/gen/JsonSourceGenerator.Parser.cs

```diff
@@ -382,18 +389,8 @@ public Parser(Compilation compilation, in JsonSourceGenerationContext sourceGene
 
                 return new SourceGenerationSpec
                 {
-                    ContextGenerationSpecList = contextGenSpecList,
-                    BooleanType = _booleanType,
-                    ByteArrayType = _byteArrayType,
-                    CharType = _charType,
-                    DateTimeType = _dateTimeType,
-                    DateTimeOffsetType = _dateTimeOffsetType,
-                    GuidType = _guidType,
-                    StringType = _stringType,
-                    NumberTypes = _numberTypes,
-#if DEBUG
-                    MetadataLoadContext = _metadataLoadContext,
-#endif
+                    ContextGenerationSpecs = contextGenSpecList.ToImmutableEquatableArray(),
```

Comment:
nit: match the names
```suggestion
                    ContextGenerationSpecs = contextGenSpecs.ToImmutableEquatableArray(),
```

Reply:
Will fix in a follow-up PR.